programs/solana/programs/ics07-tendermint/src/state.rs

```diff
@@ -0,0 +1,15 @@
+use anchor_lang::prelude::*;
+use crate::types::{ClientState, ConsensusState};
+
+#[account]
+pub struct ClientData {
+    pub client_state: ClientState,
+    pub consensus_state: ConsensusState,
+    pub frozen: bool,
+}
```

Comment:
This client data doesn't make much sense to me. Why is there a consensus state here? Shouldn't it be retrieved from ConsensusState store?

Also, isn't `frozen` contained in `client_state` already? (as frozen height > 0)